src/worker/worker.c

```diff
@@ -237,6 +240,38 @@ static int finish_job(child_process *cp, int reason)
 }
 
 /*
+ * Get the parent PID from a PID
+ */
+static int get_process_parent_id(const pid_t pid, pid_t * ppid) {
+        char buffer[BUFSIZ], *s_ppid;
+        int errreading;
+        FILE *fp;
+
+        sprintf(buffer, "/proc/%d/stat", pid);
+        fp = fopen(buffer, "r");
+        if (!fp)
```

Comment:
Use `{ }` please.

Reply:
This applies to all other places too of course

Reply:
Updated. Thnx.